### PR TITLE
inclusive definition updatea

### PR DESCRIPTION
**Pull request by StefanieBier**
Updating inclusive definitions to provide clarity.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/email-threading-in-advanced-ediscovery.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ Consider an email conversation that has been going on for a while. In most cases
 
 ## What does email threading do?
 
-Email threading parses each email and deconstructs it to individual messages; each email is a chain of individual messages. Then, it analyzes all emails in the review set to determine whether an email has unique content or if the chain is wholly contained in a different email. In the end emails are divided into four categories:
+Email threading parses each email and deconstructs it to individual messages; each email is a chain of individual messages. It then analyzes all emails in the review set to determine whether an email message has unique content or if the chain (parent messages) are wholly contained in the final email. Email messages are divided into four inclusive values:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/email-threading-in-advanced-ediscovery.md]:
```suggestion
Email threading parses each email and deconstructs it to individual messages; each email is a chain of individual messages. It then analyzes all emails in the review set to determine whether an email message has unique content or if the chain (parent messages) is wholly contained in the final email. Email messages are divided into four inclusive values:
```

File: microsoft-365/compliance/email-threading-in-advanced-ediscovery.md [hunk 2]
```diff
@@ -24,13 +24,13 @@ Consider an email conversation that has been going on for a while. In most cases
...
 
-- **Inclusive**: the last message in the email has unique content, and the email has all of the attachments that were included in other emails of which the content is wholly contained in this email.
+- **Inclusive**: An inclusive email is the the final email message in an email thread and contains all previous history of that email thread.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/email-threading-in-advanced-ediscovery.md]:
```suggestion
- **Inclusive**: An inclusive email is the final email message in an email thread and contains all the previous history of that email thread.
```

File: microsoft-365/compliance/email-threading-in-advanced-ediscovery.md [hunk 3]
```diff
@@ -24,13 +24,13 @@ Consider an email conversation that has been going on for a while. In most cases
...
 
-- **Inclusive minus**: the last message in the email has unique content, but the email does not contain some of the attachments that were included in other emails of which the content is wholly contained in this email.
+- **Inclusive minus**: An email messaage is designated as Inclusive minus if there are one or more attachments associated with the specific message within the email thread. A reviewer can use the Inclisive minus value to determine which specific emails message within the thread has associated attachments.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/email-threading-in-advanced-ediscovery.md]:
```suggestion
- **Inclusive minus**: An email message is designated as inclusive minus if there are one or more attachments associated with the specific message within the email thread. A reviewer can use the inclusive minus value to determine which specific email message within the thread has associated attachments. 
```

File: microsoft-365/compliance/email-threading-in-advanced-ediscovery.md [hunk 4]
```diff
@@ -24,13 +24,13 @@ Consider an email conversation that has been going on for a while. In most cases
...
 
-- **Inclusive**: the last message in the email has unique content, and the email has all of the attachments that were included in other emails of which the content is wholly contained in this email.
+- **Inclusive**: An inclusive email is the the final email message in an email thread and contains all previous history of that email thread.
 
-- **Inclusive minus**: the last message in the email has unique content, but the email does not contain some of the attachments that were included in other emails of which the content is wholly contained in this email.
+- **Inclusive minus**: An email messaage is designated as Inclusive minus if there are one or more attachments associated with the specific message within the email thread. A reviewer can use the Inclisive minus value to determine which specific emails message within the thread has associated attachments. 
 
-- **Inclusive copy**: an exact copy of an inclusive/inclusive minus email
+- **Inclusive copy**: An email message is considered an Inclusive copy if it is an exact copy of an inclusive/inclusive minus email.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/email-threading-in-advanced-ediscovery.md]:
```suggestion
- **Inclusive copy**: An email message is considered an inclusive copy if it is an exact copy of an inclusive/inclusive minus email. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @StefanieBier, kindly commit the suggested changes. Many thanks.

**Comment by markjjo** [comment]
@StefanieBier I have reviewed and revised updates and merged this PR. Should be live by end of day

**Pull request merged**